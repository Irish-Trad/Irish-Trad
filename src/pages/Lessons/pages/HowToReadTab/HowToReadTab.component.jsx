import React from 'react'

const HowToReadTab = () => <div>HowToReadTab</div>

export default HowToReadTab
