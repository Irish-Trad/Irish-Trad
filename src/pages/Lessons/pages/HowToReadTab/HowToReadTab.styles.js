import styled from 'styled-components'

import { Link } from 'react-router-dom'

export const MainContainer = styled.main`
  max-width: 960px;
  margin: auto;
  padding: 10px;
`

export const H1 = styled.h1`
  text-align: center;
  font-family: meathflfregular;
  font-size: 55px;
  margin: 10px;
`

export const H2 = styled.h2`
  font-size: 23px;
  font-weight: bold;
  margin: 10px 0px;
`

export const ButtonContainer = styled.div`
  display: flex;
  flex-flow: row wrap;
  justify-content: space-evenly;
`

export const Button = styled.button`
  color: white;
  background: #006400;
  border: none;
  border-radius: 12px;
  font-size: 16px;
  padding: 10px;
  cursor: pointer;
`

export const InternalLink = styled(Link)``

export const Text = styled.p`
  font-size: 18px;
`

export const Figure = styled.figure`
  display: flex;
  flex-direction: column;
  align-items: center;
`

export const Image = styled.img`
  width: 100%;
`

export const Figcaption = styled.figcaption`
  text-align: center;
`

export const ExternalLink = styled.a``
