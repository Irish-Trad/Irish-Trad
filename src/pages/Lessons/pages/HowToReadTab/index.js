export { default } from "./HowToReadTab.component";
