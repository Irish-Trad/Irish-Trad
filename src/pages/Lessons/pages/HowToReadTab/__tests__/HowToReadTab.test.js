import React from 'react'
import { render, cleanup } from '@testing-library/react'
import HowToReadTab from '../HowToReadTab.component'
import { HashRouter } from 'react-router-dom'

const renderComponent = () =>
  render(
    <HashRouter>
      <HowToReadTab />
    </HashRouter>
  )

describe('HowToReadTab', () => {
  afterEach(() => {
    cleanup()
  })

  it('Snapshot', () => {
    const snapshot = renderComponent()

    expect(snapshot).toMatchSnapshot()
  })

  it('Title of the page is How to read Tablature', () => {
    const { getByText } = renderComponent()

    expect(getByText('How to read Tablature')).toBeInTheDocument()
  })
})
