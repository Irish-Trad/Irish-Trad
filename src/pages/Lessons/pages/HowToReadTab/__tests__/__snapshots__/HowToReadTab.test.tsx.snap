// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowToReadTab Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="sc-bdVaJa dPGDrt"
      >
        How to read Tablature
      </h1>
      <div
        class="sc-htpNat cmxFXV"
      >
        <a
          class="sc-ifAKCX cFlEyZ"
          href="#/lessons"
        >
          <button
            class="sc-bxivhb cOhjdj"
          >
            Return to Lessons Page
          </button>
        </a>
      </div>
      <h2
        class="sc-bwzfXH lokaro"
      >
        Introduction:
      </h2>
      <p
        class="sc-EHOje cQMefO"
      >
        Tablature is a form of music notation designed specifically for fretted instruments such as the Irish Bouzouki. Tablature is regarded as one of the quickest methods for teaching people how to play music. The word tablature is normally shortened to just TAB.
      </p>
      <h2
        class="sc-bwzfXH lokaro"
      >
        How to:
      </h2>
      <p
        class="sc-EHOje cQMefO"
      >
        Figure 1 shows two forms of music notation (standard notation and tablature). The highlighted section is the tablature and understanding how to read this information will be the focus of the lesson.
      </p>
      <figure
        class="sc-bZQynM bsmBBS"
      >
        <a
          class="sc-dnqmqq bDsesi"
          href="assets/high-quality/TAB_Lesson/TAB02.jpg"
          target="_blank"
        >
          <img
            alt="Standard Notation & Tablature"
            class="sc-gzVnrw lfdfns"
            src="assets/low-quality/TAB_Lesson/TAB02.jpg"
            title="Standard Notation & Tablature"
          />
        </a>
        <figcaption
          class="sc-htoDjs dLESzS"
        >
          Figure 1: Standard Notation & Tablature
        </figcaption>
      </figure>
      <p
        class="sc-EHOje cQMefO"
      >
        To read tablature you have to picture that you are looking directly down on the strings of the instrument. Every string of the instrument is represented by a horizontal line. Instruments that have 4 strings (2 sets of 4) such as the Irish Bouzouki or Mandolin are represented by 4 horizontal lines.
      </p>
      <p
        class="sc-EHOje cQMefO"
      >
        Figure 2 shows the direction that the strings are read. In this example every horizontal line is labelled with the name of the string in standard tuning.
      </p>
      <figure
        class="sc-bZQynM bsmBBS"
      >
        <a
          class="sc-dnqmqq bDsesi"
          href="assets/high-quality/TAB_Lesson/TAB04.jpg"
          target="_blank"
        >
          <img
            alt="String Direction"
            class="sc-gzVnrw lfdfns"
            src="assets/low-quality/TAB_Lesson/TAB04.jpg"
            title="String Direction"
          />
        </a>
        <figcaption
          class="sc-htoDjs dLESzS"
        >
          Figure 2: String Direction
        </figcaption>
      </figure>
      <p
        class="sc-EHOje cQMefO"
      >
        Numbers presented along the horizontal line instructs you of what fret to play. In Figure 3 the highlighted note is the number 0 (open string). As the horizontal line corresponds to the G string this means that you play an open G string to play this highlighted note.
      </p>
      <figure
        class="sc-bZQynM bsmBBS"
      >
        <a
          class="sc-dnqmqq bDsesi"
          href="assets/high-quality/TAB_Lesson/TAB05.jpg"
        >
          <img
            alt="Single Note & Open String"
            class="sc-gzVnrw lfdfns"
            src="assets/low-quality/TAB_Lesson/TAB05.jpg"
            title="Single Note & Open String"
          />
        </a>
        <figcaption
          class="sc-htoDjs dLESzS"
        >
          Figure 3: Single Note & Open String
        </figcaption>
      </figure>
      <p
        class="sc-EHOje cQMefO"
      >
        Figure 4 highlights a section that shows how to play two notes at the same time. Numbers that are presented directly above and below each other mean they are played simultaneously. The highlighted section instructs you to play the 2nd fret of the D string and the open G string.
      </p>
      <figure
        class="sc-bZQynM bsmBBS"
      >
        <a
          class="sc-dnqmqq bDsesi"
          href="assets/high-quality/TAB_Lesson/TAB06.jpg"
          target="_blank"
        >
          <img
            alt="More Than One Note"
            class="sc-gzVnrw lfdfns"
            src="assets/low-quality/TAB_Lesson/TAB06.jpg"
            title="More Than One Note"
          />
        </a>
        <figcaption
          class="sc-htoDjs dLESzS"
        >
          Figure 4: More Than One Note
        </figcaption>
      </figure>
      <p
        class="sc-EHOje cQMefO"
      >
        The highlighted section in Figure 5 shows how to play a note on every string at the same time. Doing this results in a chord. For this the G and D string are played open while the A string is fretted at 2nd fret and the E string is the 3rd fret. Strumming all the strings in a swift motion causes them to ring simultaneously. This produces a G Major chord.
      </p>
      <figure
        class="sc-bZQynM bsmBBS"
      >
        <a
          class="sc-dnqmqq bDsesi"
          href="assets/high-quality/TAB_Lesson/TAB07.jpg"
          target="_blank"
        >
          <img
            alt="Full Chord"
            class="sc-gzVnrw lfdfns"
            src="assets/low-quality/TAB_Lesson/TAB07.jpg"
            title="Full Chord"
          />
        </a>
        <figcaption
          class="sc-htoDjs dLESzS"
        >
          Figure 5: Full Chord
        </figcaption>
      </figure>
      <p
        class="sc-EHOje cQMefO"
      >
        Figure 6 is a series of single notes played in order. This actually performs the G Major scale. Information about chords and scales will be provided in future lessons.
      </p>
      <figure
        class="sc-bZQynM bsmBBS"
      >
        <a
          class="sc-dnqmqq bDsesi"
          href="assets/high-quality/TAB_Lesson/TAB08.jpg"
          target="_blank"
        >
          <img
            alt="Playing Through A Scale"
            class="sc-gzVnrw lfdfns"
            src="assets/low-quality/TAB_Lesson/TAB08.jpg"
            title="Playing Through A Scale"
          />
        </a>
        <figcaption
          class="sc-htoDjs dLESzS"
        >
          Figure 6: Playing Through A Scale
        </figcaption>
      </figure>
      <h2
        class="sc-bwzfXH lokaro"
      >
        Summary:
      </h2>
      <p
        class="sc-EHOje cQMefO"
      >
        Tablature (TAB) presents each string of an instrument on a horizontal line and instructs the musician of which fret to play by placing numbers along those horizontal lines. The thickest and lowest pitched string is at the bottom of the tablature while the thinnest and highest pitched string is at the top. Notes that are directly above and below each other on the horizontal line are played simeltaneously.
      </p>
      <p
        class="sc-EHOje cQMefO"
      >
        While not showcased in this lesson tablature is also able represent (using symbols) a variety of techniques along the horizontal lines such as sliding between notes, hammer-ons and pull-offs, trills and bends.
      </p>
    </div>
  </body>,
  "container": <div>
    <h1
      class="sc-bdVaJa dPGDrt"
    >
      How to read Tablature
    </h1>
    <div
      class="sc-htpNat cmxFXV"
    >
      <a
        class="sc-ifAKCX cFlEyZ"
        href="#/lessons"
      >
        <button
          class="sc-bxivhb cOhjdj"
        >
          Return to Lessons Page
        </button>
      </a>
    </div>
    <h2
      class="sc-bwzfXH lokaro"
    >
      Introduction:
    </h2>
    <p
      class="sc-EHOje cQMefO"
    >
      Tablature is a form of music notation designed specifically for fretted instruments such as the Irish Bouzouki. Tablature is regarded as one of the quickest methods for teaching people how to play music. The word tablature is normally shortened to just TAB.
    </p>
    <h2
      class="sc-bwzfXH lokaro"
    >
      How to:
    </h2>
    <p
      class="sc-EHOje cQMefO"
    >
      Figure 1 shows two forms of music notation (standard notation and tablature). The highlighted section is the tablature and understanding how to read this information will be the focus of the lesson.
    </p>
    <figure
      class="sc-bZQynM bsmBBS"
    >
      <a
        class="sc-dnqmqq bDsesi"
        href="assets/high-quality/TAB_Lesson/TAB02.jpg"
        target="_blank"
      >
        <img
          alt="Standard Notation & Tablature"
          class="sc-gzVnrw lfdfns"
          src="assets/low-quality/TAB_Lesson/TAB02.jpg"
          title="Standard Notation & Tablature"
        />
      </a>
      <figcaption
        class="sc-htoDjs dLESzS"
      >
        Figure 1: Standard Notation & Tablature
      </figcaption>
    </figure>
    <p
      class="sc-EHOje cQMefO"
    >
      To read tablature you have to picture that you are looking directly down on the strings of the instrument. Every string of the instrument is represented by a horizontal line. Instruments that have 4 strings (2 sets of 4) such as the Irish Bouzouki or Mandolin are represented by 4 horizontal lines.
    </p>
    <p
      class="sc-EHOje cQMefO"
    >
      Figure 2 shows the direction that the strings are read. In this example every horizontal line is labelled with the name of the string in standard tuning.
    </p>
    <figure
      class="sc-bZQynM bsmBBS"
    >
      <a
        class="sc-dnqmqq bDsesi"
        href="assets/high-quality/TAB_Lesson/TAB04.jpg"
        target="_blank"
      >
        <img
          alt="String Direction"
          class="sc-gzVnrw lfdfns"
          src="assets/low-quality/TAB_Lesson/TAB04.jpg"
          title="String Direction"
        />
      </a>
      <figcaption
        class="sc-htoDjs dLESzS"
      >
        Figure 2: String Direction
      </figcaption>
    </figure>
    <p
      class="sc-EHOje cQMefO"
    >
      Numbers presented along the horizontal line instructs you of what fret to play. In Figure 3 the highlighted note is the number 0 (open string). As the horizontal line corresponds to the G string this means that you play an open G string to play this highlighted note.
    </p>
    <figure
      class="sc-bZQynM bsmBBS"
    >
      <a
        class="sc-dnqmqq bDsesi"
        href="assets/high-quality/TAB_Lesson/TAB05.jpg"
      >
        <img
          alt="Single Note & Open String"
          class="sc-gzVnrw lfdfns"
          src="assets/low-quality/TAB_Lesson/TAB05.jpg"
          title="Single Note & Open String"
        />
      </a>
      <figcaption
        class="sc-htoDjs dLESzS"
      >
        Figure 3: Single Note & Open String
      </figcaption>
    </figure>
    <p
      class="sc-EHOje cQMefO"
    >
      Figure 4 highlights a section that shows how to play two notes at the same time. Numbers that are presented directly above and below each other mean they are played simultaneously. The highlighted section instructs you to play the 2nd fret of the D string and the open G string.
    </p>
    <figure
      class="sc-bZQynM bsmBBS"
    >
      <a
        class="sc-dnqmqq bDsesi"
        href="assets/high-quality/TAB_Lesson/TAB06.jpg"
        target="_blank"
      >
        <img
          alt="More Than One Note"
          class="sc-gzVnrw lfdfns"
          src="assets/low-quality/TAB_Lesson/TAB06.jpg"
          title="More Than One Note"
        />
      </a>
      <figcaption
        class="sc-htoDjs dLESzS"
      >
        Figure 4: More Than One Note
      </figcaption>
    </figure>
    <p
      class="sc-EHOje cQMefO"
    >
      The highlighted section in Figure 5 shows how to play a note on every string at the same time. Doing this results in a chord. For this the G and D string are played open while the A string is fretted at 2nd fret and the E string is the 3rd fret. Strumming all the strings in a swift motion causes them to ring simultaneously. This produces a G Major chord.
    </p>
    <figure
      class="sc-bZQynM bsmBBS"
    >
      <a
        class="sc-dnqmqq bDsesi"
        href="assets/high-quality/TAB_Lesson/TAB07.jpg"
        target="_blank"
      >
        <img
          alt="Full Chord"
          class="sc-gzVnrw lfdfns"
          src="assets/low-quality/TAB_Lesson/TAB07.jpg"
          title="Full Chord"
        />
      </a>
      <figcaption
        class="sc-htoDjs dLESzS"
      >
        Figure 5: Full Chord
      </figcaption>
    </figure>
    <p
      class="sc-EHOje cQMefO"
    >
      Figure 6 is a series of single notes played in order. This actually performs the G Major scale. Information about chords and scales will be provided in future lessons.
    </p>
    <figure
      class="sc-bZQynM bsmBBS"
    >
      <a
        class="sc-dnqmqq bDsesi"
        href="assets/high-quality/TAB_Lesson/TAB08.jpg"
        target="_blank"
      >
        <img
          alt="Playing Through A Scale"
          class="sc-gzVnrw lfdfns"
          src="assets/low-quality/TAB_Lesson/TAB08.jpg"
          title="Playing Through A Scale"
        />
      </a>
      <figcaption
        class="sc-htoDjs dLESzS"
      >
        Figure 6: Playing Through A Scale
      </figcaption>
    </figure>
    <h2
      class="sc-bwzfXH lokaro"
    >
      Summary:
    </h2>
    <p
      class="sc-EHOje cQMefO"
    >
      Tablature (TAB) presents each string of an instrument on a horizontal line and instructs the musician of which fret to play by placing numbers along those horizontal lines. The thickest and lowest pitched string is at the bottom of the tablature while the thinnest and highest pitched string is at the top. Notes that are directly above and below each other on the horizontal line are played simeltaneously.
    </p>
    <p
      class="sc-EHOje cQMefO"
    >
      While not showcased in this lesson tablature is also able represent (using symbols) a variety of techniques along the horizontal lines such as sliding between notes, hammer-ons and pull-offs, trills and bends.
    </p>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
